src/controllers/queryRunner.ts

```diff
@@ -561,6 +561,27 @@ export default class QueryRunner {
         return headers;
     }
 
+    public async copyHeaders(batchId: number, resultId: number): Promise<void> {
+        let headers = this.getColumnHeaders(batchId, resultId, {
+            fromCell: 0,
+            fromRow: 0,
+            toCell: this.batchSets[batchId].resultSetSummaries[resultId]
+                .columnInfo.length,
+            toRow: this.batchSets[batchId].resultSetSummaries[resultId]
+                .columnInfo.length,
+        });
+        let headerString = headers.join("\t");
+        let oldLang: string;
```

Comment:
Maybe use vscode api to avoid handling OS specific cases

https://github.com/microsoft/vscode-mssql/blob/6c15ba9821f826271e85efa2458cdb4fc8cc6515/src/tableDesigner/tableDesignerWebviewController.ts#L362